packages/app/src/systems/Chains/eth/hooks/useEthAccountConnection.tsx

```diff
@@ -21,11 +21,14 @@ export function useEthAccountConnection(props?: {
   const { address, isConnected } = useAccount();
   const { data: ensName } = useEnsName({ address });
   const { data: ensAvatar } = useEnsAvatar({ name: address });
-  const { data: balance } = useBalance({ address, token: erc20Address });
+  const { data: balance } = useBalance({
+    address,
+    token: erc20Address !== NativeAssetId ? erc20Address : undefined,
+  });
   const publicClient = usePublicClient();
   const { data: walletClient } = useWalletClient();
   const { asset } = useAsset({
-    address: erc20Address && isAddress(erc20Address) ? erc20Address : undefined,
+    address: erc20Address,
```

Comment:
why to remove the security validation ?

Reply:
isAddress does not work on the NativeAssetId